sno/working_copy/postgis.py

```diff
@@ -46,17 +46,38 @@ def adapt_geometry_from_db(g, dbcur):
 
 def adapt_timestamp_from_db(t, dbcur):
     # TODO - revisit timezones.
-    if isinstance(t, str):
-        # This makes postgis timestamps behave more like GPKG ones.
-        return t.replace(" ", "T") + "Z"
-    return t
+    if t is None:
+        return t
+    # This makes postgis timestamps behave more like GPKG ones.
```

Comment:
Postgis timestamps-with-time-zones strings could be `2020-11-13T10:00:00+13:00` though? Seems like suffixing `Z` is too simplistic?